### PR TITLE
feat: add audit log

### PR DESCRIPTION
**Pull request by elysee15**
Fixes SC-1321

### CODE REVIEW

**Comment by bbengfort** [comment]
@elysee15 would you mind rebasing this with `main` please? Also are you ready for me to review this? Sorry I've been getting behind on the PR reviews.

File: web/gds-admin-ui/src/pages/app/details/index.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const VaspDetails = () => {
                         </Col>
                         <Col md={6} xl={4} xxl={4}>
                             <Contact data={vasp?.vasp?.contacts} verifiedContact={vasp?.verified_contacts} />
-                            <AuditLog data={vasp?.vasp?.extra} />
+                            <AuditLog data={vasp?.vasp?.audit_log} />
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/details/index.js:55]: @elysee15 it looks like the `AuditLog` got merged into main? 

I did try running this locally and I'm not seeing an AuditLog widget between the contact and the certificate details. I am getting audit log responses back from the API. Am I doing something wrong?

Comment (elysee15) [reply web/gds-admin-ui/src/pages/app/details/index.js:55]: I changed how I accessed to the `audit.log`. Is it a `extra` property or `vasp` property ?

**Review by bbengfort — APPROVED** [review]
@elysee15 looks good - cheers! I'm merging so I can get this into the deployment.

**Pull request merged**